asm/bss_8001C4D0.s

```diff
@@ -2082,7 +2082,7 @@ glabel D_8018EDEE
 glabel gDebugMenuSelection
 .skip 1
 
-glabel gPlayerCountSelection2 # 0x8018EDF0
+glabel gMenuState # 0x8018EDF0
```

Comment:
I don't really know what to name this but its not related to player count at all. It only coincidentally matches the human player count on the main menu, its not used for that purpose though

Reply:
It looks like its value corresponds to what is selected in the Controller Pak Menu and func_800B1C90 is handling what to do for controller inputs depending on the selection.

Reply:
That's part of it, its also has values that only show up when you delete an entry from the controller pack. It also appears to be used in the main menu, indicating which player-count mode you've chosen, which is probably why it was original named "PlayerCountSelection" because it does sort of look like that's what its tracking

Reply:
Is the symbol used for two purposes? Or is the player-count theory wrong?
If this is for controllerPak. Would it make sense for everything to be named something like:
`gControllerPakState`?

Reply:
It appears to be used in places other than the controller pak menu. Its just that it probably got its original name because if all you look at in the main menu it does look like its tracking player count. That's not what its doing, but I can see why it would be identified that way.

Reply:
I see D_8018EDF3 matching the player count on the main menu, but D_8018EDF0 is only a byte